packages/react-aria-components/docs/Popover.mdx

```diff
@@ -424,16 +411,13 @@ The `className` and `style` props also accept functions which receive states for
 Popovers also support entry and exit animations via states exposed as data attributes and render props. `Popover` will automatically wait for any exit animations to complete before it is removed from the DOM.
```

Comment:
I think the explanation in the CSS transition page styling is good, it might be worthwhile linking to it from the Popover/Tooltip/Modal/etc pages and/or via the JSDOC of the `isEntering`/`isExiting` props so it is more discoverable.